tilequeue/process.py

```diff
@@ -556,16 +562,56 @@ def format_coord(
 # the output.
 def process_coord(coord, nominal_zoom, feature_layers, post_process_data,
                   formats, unpadded_bounds, cut_coords, buffer_cfg,
-                  output_calc_spec, scale=4096, log_fn=None, max_zoom_with_changes=16):
+                  output_calc_spec, scale=4096, log_fn=None,
```

Comment:
I stared at this line for like 10 seconds before I realized there was no change

Reply:
We should run our autoformatter on the whole repo

Reply:
sorry, it was an unintended change, but it does pass the linter too. anyway, I reverted the change.